src2/com/piseth/java/school/service/PersonService2.java

```diff
@@ -0,0 +1,22 @@
+package com.piseth.java.school.service;
+
+import java.sql.Connection;
+import java.util.List;
+import java.util.Map;
+
+import com.piseth.java.school.model.Person2;
+
+public interface PersonService2 {
+	//-------------------------------------Exercise No.3-------------------------------------------//
+		Map<String, Integer> countNumberOfPeopleByGender(Connection conn);
```

Comment:
There is no need to put Connection object in every method. You should put connection object as field member in PersonServiceImpl.